src/WebJobs.Extensions/Internals/CommonCollectorBinding.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
```

Comment:
@MikeStall @brettsam How far away do you guys think all these binding helpers are from being to the point where we could just make them public in the core SDK? The core SDK will be in prerelease builds for a while, so we'd have some time to solidify the interfaces, etc.

If we think shortly after build we could have them all in shape enough to release, them perhaps we just expose them from the core SDK now and iterate on them there. That would save us from the ugliness that we already knew we'd see in copying these files in, but is all the more poignant now that I'm CR it :)